lit/setup-and-operations/install.lit

```diff
@@ -42,17 +42,17 @@ page.
   you might want to just kick the tires a bit and run everything at once. This
   can be achieved with the \code{quickstart} command:
 
-  \codeblock{bash}{{
-  concourse quickstart \\
-    --basic-auth-username myuser \\
-    --basic-auth-password mypass \\
-    --external-url http://my-ci.example.com \\
+  \codeblock{bash}{{{
+  concourse quickstart \
+    --add-local-user myuser:mypass \
```

Comment:
The name of this flag throws me off a bit - is it doing a mutation (literally adding a user somewhere) or just configuring the user setup (which I see is global, and then you grant the user access to the main team separately)? If I stop passing the flag, is the user still there?